scaffolder-templates/basic-workflow/template.yaml

```diff
@@ -335,6 +335,19 @@ spec:
         repoUrl: github.com?owner=${{ parameters.orgName }}&repo=${{ parameters.repoName }}-gitops
         defaultBranch: main
         sourcePath: gitops
+
+    # TODO find out how to skip this if this if the argocd scaffolder plugin isn't installed
+    - id: create-argocd-resources
+      if: ${{ parameters.CI == 'tekton_argocd' }}
+      name: Create ArgoCD Resources
+      action: argocd:create-resources
+      input:
+        appName: ${{ parameters.component_id }}-bootstrap
+        argoInstance: main
```

Comment:
Is the `main` instance name the same as the one define in the plugin configuraiton at
https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator/templates/rhdh-operator.yaml#L185 ?